### DISCUSSION TITLE
Twilio bill increasing by 800% with ASv1.5.0

### DISCUSSION THREAD

**Discussion by arthurgousset** [celo-org/social-connect#226] [Help (Q&A)]
Reflecting a question asked by user `mbay2002 |  VCAP` on [Discord](https://discord.com/channels/600834479145353243/654086275900375041/973613943169884241): 

> April was the first full month that we ran attestation-service-v1.5.0, and we saw our Twilio bill increased fairly significantly.... digging into it, there were more calls to the "authy" twilio API than previously. Anybody else notice this?  I think this is the "twilioverify" service in the config file.
> 
> 800%. Would be nice to know if others are seeing something similar.

**Comment by aaronmboyd** [comment]
I'm seeing similar looking increases in billing.

Dec 2022 - $8.99
Jan 2022 - $10.75
Feb 2022 - $8.28
March 2022 - $22.46 <=== Authy and Verify now appear in billing from here
April 2022 - $30.06
